pkg/pfcpsim/session/far_builder.go

```diff
@@ -98,10 +108,13 @@ func (b *farBuilder) BuildFAR() *ie.IE {
 
 	far := createFunc(
 		ie.NewFARID(b.farID),
-		ie.NewApplyAction(b.applyAction),
 		fwdParams,
 	)
 
+	for _, action := range b.actions {
+		far.Add(ie.NewApplyAction(action))
```

Comment:
How does the `ie` library behave in this case? Isn't the previous ApplyAction IE overwritten ?

Reply:
It doesn't seem so. I wrote a test adding several new actions and the length is different. 
Note the difference between the expected and the input. I made the test fail on purpose

![image](https://user-images.githubusercontent.com/26747011/154241747-be620225-c053-4bc5-83a5-8168c5e4afcd.png)